src/cmdstan-guide/laplace_sample_config.Rmd

```diff
@@ -0,0 +1,100 @@
+# Laplace sampling
+
+The `laplace` method produces a sample from a Laplace approximation
+centered at the mode of a distribution.
+If the mode is a maximum a posteriori (MAP) estimate,
+the samples provides an estimate of the standard deviation
```

Comment:
It would be good to explicitly mention here the effect of the constraints. If there are constrained parameters, the Laplace approximation works best in the unconstrained space. The normal approximation is then centered at the mode in the unconstrained space, but the implemented method transforms the normal approximation sample to the constrained space before writing them to csv. Due to the transformation, the samples can provide more than just the standard deviation estimate.

Reply:
this discussion should go in the reference manual algorithms section, and we should link to that from here.

I'm confused by what you say - the `laplace_sample` algorithm always computes on the unconstrained space, and outputs on the constrained scale - that is the way that all the Stan inference algorithms work, so why mention this?

> Due to the transformation, the samples can provide more than just the standard deviation estimate.

what additional inforamation is provided?   a better estimate of the mode as well?

Reply:
apologies - my confusion here is that while the  `stan::services` method `laplace_sample` takes its inputs on the unconstrained scale, the CmdStan interface takes its inputs on the constrained scale, and then does the unconstrain before calling the `laplace_sample` method.  this is an implementation detail that the user doesn't need to know about.

Reply:
added discussion of this to the Laplace section in the reference manual.

Reply:
I would still change this also here, e.g.
"The `laplace` method produces a sample from a normal approximation
centered at the mode of a distribution in the unconstrained space.
If the mode is a maximum a posteriori (MAP) estimate,
the samples provide an estimate of the mean and standard deviation
of the posterior distribution."

In case of constraints, in general the posterior mode in the unconstrained space doesn't correspond to the mean (nor mode) in the constraint space, and thus the sample is needed also to infer the mean.